Bio/motifs/minimal.py

```diff
@@ -98,21 +98,29 @@ def _read_background(record, handle):
     """Read background letter frequencies (PRIVATE)."""
     for line in handle:
         if line.startswith("Background letter frequencies"):
+            background_freqs = []
+            while True:
+                line = next(handle).rstrip()
```

Comment:
I believe you can simply use `for line in handle` here

Reply:
Fixed.